package.json

```diff
@@ -19,15 +19,15 @@
     "url": "https://github.com/ventureharbour/ventureneering/issues"
   },
   "scripts": {
-    "dev": "docusaurus start",
+    "dev": "docusaurus start --port 3333",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "generate-post": "node generatePost.js"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.15",
-    "@docusaurus/preset-classic": "2.0.0-beta.15",
+    "@docusaurus/core": "2.0.0-beta.18",
+    "@docusaurus/preset-classic": "2.0.0-beta.18",
```

Comment:
booo, also did the update work? I always get so many errors when updating hence why I locked it to a specific beta?

Reply:
ya, I just pulled and tried run dev and it's very unhappy

Reply:
works fine for me, if I use node version 14.16.1. I can seperate the post from the build into two PRs.